docs/content.zh/docs/connectors/datastream/pubsub.md

```diff
@@ -28,7 +28,7 @@ under the License.
 
 这个连接器可向 [Google Cloud PubSub](https://cloud.google.com/pubsub) 读取与写入数据。添加下面的依赖来使用此连接器:
 
-{{< connector_artifact flink-connector-pubsub 3.0.0 >}}
```

Comment:
This looks like a typo error.

Reply:
Good catch, this is a previously existing issue but I will fix it in a hot-fix commit.